server/src/main/java/org/candlepin/pki/impl/BouncyCastlePKIUtility.java

```diff
@@ -78,8 +78,7 @@
  * objects in PEM format (for saving to the db or sending to the client), and
  * decode raw ASN.1 DER values (as read from a Certificate/CRL).
  *
- * All code that imports bouncycastle should live either in this module,
- * or in {@link BouncyCastlePKIReader}
+ * All code that imports bouncycastle should live in this module.
```

Comment:
My goal in this PR is not to eliminate all usages of BouncyCastle, but to eliminate usages of `PEMParser` and `PemReader` since these are very high level classes that use a lot of BouncyCastle internals.  Even still, for PKCS1 style keys, we're continuing to use the BouncyCastle ASN1 parsing tools.